setup.py

```diff
@@ -1,14 +1,20 @@
 from setuptools import find_packages, setup
 
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
```

Comment:
```suggestion
long_description = open("README.md").read()
```
python got smarter and I like this new shorter way of reading in files.
this could even be inline on line 16. it's not important though.